runtime/near-vm-runner/src/imports.rs

```diff
@@ -155,7 +155,7 @@ imports! {
         amount_ptr: u64,
         gas: u64
     ] -> []>,
-    #["protocol_feature_function_call_weight", FunctionCallWeight] promise_batch_action_function_call_weight<[
```

Comment:
This should say:

```
#[FunctionCallWeight] promise_batch_action_function_call_weight
```

we still want to guard this import *at runtime*, so it isn't accessible on old protocol versions

Reply:
oh oops, of course